ludwig/features/binary_feature.py

```diff
@@ -47,6 +47,12 @@ class BinaryFeatureMixin:
         'fill_value': 0
     }
 
+    preprocessing_schema = {
+        'missing_value_strategy': {'type': 'string', 'enum': MISSING_VALUE_STRATEGY_OPTIONS},
+        'fill_value': {'type': 'integer'},
```

Comment:
this may be also True, False and a string in case of binry features with a vocab like Yes/No

Reply:
Added all string possibilities.